### ISSUE TITLE
NPM Module

### ISSUE THREAD

**Issue by bigggprogrammerboy** [joshmarinacci/voxeljs-next#34]
I got really excited when I saw someone was updating voxeljs, because I have been working on a from scratch equivalent as the base of a new game, but, unless I'm crazy, there's no good way to consume this awesome library, other than cloning the files into my own repo.

Could we spool up an npm package to make this easier to use in other projects? Happy to do the work, and looking forward to tackling the LOD and AO issues once I get this up and running in my own project.

**Comment by joshmarinacci** [comment]
Yeah, it really needs an npm module and possibly a compile/rollup stage. I've just been too busy to do it. I would love some help.

**Comment by bigggprogrammerboy** [comment]
Any preference in tooling?

**Comment by joshmarinacci** [comment]
no. what are your thoughts?

**Comment by bigggprogrammerboy** [comment]
Parcel?

**Comment by Rizzist** [comment]
Any updates on an NPM Module? Looking forward to using voxeljs-next in my own project.

**Comment by joshmarinacci** [comment]
I haven't yet, but I was just laid off so I should have more time in the coming weeks. I'll add it to my list.

**Comment by Rizzist** [comment]
Nice! If you do finish this by August 31st, I would love to have a look at examples, I am planning on using with socket.io to make a genuine multiplayer game. In the meantime I am going to try this: https://threejsfundamentals.org/threejs/lessons/threejs-voxel-geometry.html
